scripts/osx/testTemplate.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+OF_DIR=`pwd`;
+echo $OF_DIR;
+TEST_DIR="$OF_DIR/scripts/osx"
+cd $TEST_DIR
```

Comment:
although, under the danger or starting to bikeshed now, I think it would even make more sense to have one big script to orchestrate all the testing, which selects based on given arguments, e.g. like this:

```
matrix:
  - NAME="osx"
  - NAME="ios"
  - NAME="doc"
before_install:
  "if [ \"$NAME\" == \"doc\" ]; then ./scripts/ci_script.sh $NAME before_install; fi"
script:
  ./scripts/ci_script.sh $NAME
after_success:
  "if [ \"$NAME\" == \"doc\" ]; then ./scripts/ci_script.sh $NAME after_success; fi"
```

this is most clear, all the uninteresting bookkeeping is in the script, the script can also be called by a user locally like `ci_script.sh ios` (kinda like we use apothecary now), and the env var list is not polluted/can easily be matrixed with OSes etc. later.
@danoli3 @arturoc what do you think?